### ISSUE TITLE
Semen, Hatching Eggs, etc.

### ISSUE THREAD

**Issue by mkm1879** [tracefirst/usaha_committee#24]
We will need to address the odd CVI that deals with things other than live, born animals.  CVIs are routinely used for semen, hatching eggs, etc.

Hatching eggs could have a negative age, I guess!  Semen is a little different.  

Really, I think a field is needed to call out these things that are not specific living animals.  Or, modify our scope to make this out of scope, but that might limit utility.

**Comment by mkm1879** [comment]
I've recently attempted to implement products in the standard using GroupLot and found that it works except for the need of an attribute to identify the product.  

I ended up implementing as if it was a group of animals and then bulk updating the database after the fact.  The only missing information was the "product"

I would suggest an optional attribute on the GroupLot element to hold the product.  Values I know of are "Eggs" and "Semen".  If this attribute is omitted, "Live Animals" would be assumed.  I suppose we could include that in the list for systems that wanted to be explicit.
